### DISCUSSION TITLE
How do i join to a table multiple times?

### DISCUSSION THREAD

**Discussion by happymacarts** [rappasoft/laravel-livewire-tables#1459] [Q&A]
I have an "invited_by" column on my user table and an invitor hasOne relationship on my user model.
how do I retrieve the relationship without getting the error

> SQLSTATE[42000]: Syntax error or access violation: 1066 Not unique table/alias: 'users'

Component
`Column::make('Invited By','invitor.name') ,`

Model
` public function invitor(): HasOne
    {
        return $this->hasOne(User::class, 'id', 'invited_by');
    }
`

**Comment by carbtest** [comment]
Wondered if anyone has a solution on how to use the same table twice in a builder query without getting the errors?

**Reply by lrljoe** [reply]
v3 should cater for this.  Whether it makes it as a backport into v2 - not sure yet!
